vite.config.js

```diff
@@ -53,11 +46,9 @@ module.exports = ({ command, mode }) => {
   const publicDir = mode == "production" ? publicDirProd : publicDirDev;
   return {
     // TODO Remove this if we get EnvironmentPlugin to work.
-    define: {
-      'process.env.CATS_EFFECT_TRACING_MODE': "none"
-    },
     root: "explore/src/main/webapp",
     publicDir: publicDir,
+    envPrefix: ["VITE_", "CATS_EFFECT_"],
```

Comment:
This is a good idea, it should carry over environment variables into the app. I think it's probably redundant with lines 15-17 (in case vite honors it).